Harvest.Web/ClientApp/src/Projects/ProjectListContainer.tsx

```diff
@@ -22,7 +22,17 @@ export const ProjectListContainer = () => {
 
   return (
     <div>
-      <Link to="/request/create">Create New</Link>
+      <div className="row justify-content-between mb-3">
+        <div className="col">
+          <h1>Projects</h1>
+        </div>
+        <div className="col text-right">
+          <a className="btn btn-sm btn-link " href="/request/create">
```

Comment:
Unfortunately you can't replace `<Link>` with `<a>` because they are not really the same thing so we'll need to put that back.  But it works just like <a> so you can put content inside and add classes to it.